daemon/logger/jsonfile/jsonfile.go

```diff
@@ -0,0 +1,60 @@
+package jsonfile
+
+import (
+	"os"
+	"sync"
+
+	"github.com/alibaba/pouch/daemon/logger"
+)
+
+// JSONLogFile is uses to log the container's stdout and stderr.
+type JSONLogFile struct {
+	mu sync.Mutex
```

Comment:
I am wondering this lock will detect what kind of critical section.
At least, in my opinion, a `f *os.File` does not need locked. This is not an instance allocated in memory and shared within several goroutine. It is specific filesystem and io resource to guarantee the concurrent writing of the file. WDYT? @fuweid

Reply:
I'm still wondering the case that if we need `file.Name()` related action. The case still needs the lock. Other cases, like `Write` and `Close`, need the lock to make sure the concurrent issue. I think we can keep the lock here. After make the container.io stable and high performance, we will remove the lock if we don't need it. WDYT?

Reply:
make sense.